sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_constants.py

```diff
@@ -27,3 +27,11 @@
 # Mime profiles
 APP_CONFIG_AI_MIME_PROFILE = "https://azconfig.io/mime-profiles/ai/"
 APP_CONFIG_AICC_MIME_PROFILE = "https://azconfig.io/mime-profiles/ai/chat-completion"
+APP_CONFIG_SNAPSHOT_REF_MIME_PROFILE = "https://azconfig.io/mime-profiles/snapshot-ref"
```

Comment:
No need to track MIME profile - just compare with SNAPSHOT_REF_CONTENT_TYPE to check if it's a snapshot reference. We do the same in .net provider: https://github.com/Azure/AppConfiguration-DotnetProvider/blob/34576ef70d303a029b7ea7ef860f12885fb8f35d/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs#L869C45-L869C57